index.html

```diff
@@ -1,170 +1,262 @@
+                <img
+                  src="./images/Rectangle 55 (1).png"
+                />Languages</a
+              >
+            </li>
+            <li><a href="#">Javascript</a></li>
+            <li><a href="#">Ruby</a></li>
+            <li><a href="#">Html</a></li>
+            <li><a href="#">Css</a></li>
+            <li class="langu">
+              <a href="#">
+                <img src="./images/Rectangle 56.png" />Frameworks</a
+              >
+            </li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby on Rails</a></li>
+            <li><a href="#">RSPec</a></li>
+            <li><a href="#">CapyBara</a></li>
+            <li><a href="#">Selenium</a></li>
+            <li class="langu">
+              <a href="#">
+                <img src="./images/Rectangle 56.png" />Skills</a
+              >
+            </li>
+            <li><a href="#">Codekit</a></li>
+            <li><a href="#">Github</a></li>
+            <li><a href="#">Codepen</a></li>
+            <li><a href="#">Gitlab</a></li>
+            <li><a href="#">Terminal</a></li>
+          </ul>
         </div>
+      </div>
+      <div class="skills"></div>
     </div>
-
-</body>
-
-</html>
\ No newline at end of file
+    <footer>
+      <form action="https://formspree.io/f/xoqbglnq" method="post">
+        <p>
+          I'm always <br>interested in hearing <br>about new projects,<br> so if
+          you'd like to <br>chat please get in touch.
+        </p>
+        <ul>
+          <li>
+            <input
+              type="text"
+              placeholder="Full Name"
+              name="user_name"
+              required
+              maxlength="30"
+            />
+          </li>
+          <hr style="border-bottom: 1px solid #DBD8D7; margin-top:10%;width: 85%;">
+          <li>
+            <input type="email" placeholder="Email" name="user_email" required />
+          </li>
+          <hr style="border-bottom: 1px solid #DBD8D7; margin-top:10%; width: 85%;">
+          <li>
+            <textarea
+              name="use-message"
+              id=""
+              maxlength="500"
+              placeholder="Enter your text here"
+              required
+            ></textarea>
+          </li>
+          <li class="button">
+            <button>Get in touch</button>
+          </li>
+        </ul>
+      </form>
+      <hr style="border-bottom: 1px solid #DBD8D7; margin-top:30%;width: 85%;">
+      <!-- icons -->
+      <section class="icons">
+        <ul class="footer-social-links">
+          <a href="#"><img src="./images/Vector f1.png" /></a>
+          <a href="#"><img src="./images/Vector-17.svg" style="margin-right: 2px;height: 17px;"><img src="./images/Vector f7.png"/></a>
+          <a href="#"><img src="./images/Vector f3.png" /></a>
+          <a href="#"><img src="./images/Vector f4.png" /></a>
+        </ul>
+        <div class="bar"></div>
+      </section>
+    </footer>
```

Comment:
##### Impressive changes you made @Tracy-miranja. But There are some minor details we need to address to get your design close to the Figma design.
- [ ] Please make sure to use the right font as defined in the Figma design. In your case please use `Inter` font family.
- [ ] Please make sure your `about` content is aligned to the center as shown in the Figma design.

### Figma design
![Screenshot 2022-12-07 203111](https://user-images.githubusercontent.com/69077061/206406387-5a1b0e50-06b6-4dce-a1be-4c6f08ae77f5.png)

### Your design
![Screenshot 2022-12-07 203211](https://user-images.githubusercontent.com/69077061/206406390-648b1736-2ed1-4a8d-bd1b-9888dd4c5689.png)